packages/storage-driver-s3/package.json

```diff
@@ -0,0 +1,44 @@
+{
+	"name": "@directus/storage-driver-s3",
+	"version": "9.21.2",
+	"type": "module",
+	"scripts": {
+		"build": "tsc --build",
+		"dev": "tsc --watch",
+		"test": "vitest run",
+		"test:watch": "vitest",
+		"test:coverage": "vitest run --coverage"
+	},
+	"description": "S3 file storage abstraction for `@directus/storage`",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/directus/directus.git",
+		"directory": "packages/storage-driver-s3"
+	},
+	"funding": "https://github.com/directus/directus?sponsor=1",
+	"license": "GPL-3.0",
+	"author": "Rijk van Zanten <rijkvanzanten@me.com>",
+	"exports": {
+		".": "./dist/index.js",
+		"./package.json": "./package.json"
+	},
+	"main": "dist/index.js",
+	"files": [
+		"dist",
+		"!**/*.d.ts?(.map)"
+	],
+	"publishConfig": {
+		"access": "public"
+	},
+	"dependencies": {
+		"@aws-sdk/client-s3": "3.224.0",
```

Comment:
Since this PR has also updated the s3 driver to use the latest v3 aws sdk, I think we can mark this to close #16059. However, we should also probably confirm this with an existing PR #16424 which aimed to do the same (and maybe close it in favor of this or carry over any implementations from there, if applicable).

Reply:
Totally! I actually reused the changes in that PR. Can't believe I forgot to put it in the PR description 🙈 (Done now!)